Cargo.toml

```diff
@@ -23,6 +23,10 @@ semver = "0.5.0"
 separator = "0.3.1"
 time = "0.1.35"
 
+[dependencies.clippy]
+optional = true
+version = "0.0.118"
```

Comment:
I didn't think of this until now, but clippy might be one of those rare cases in which we do actually want `version = "*"`; it seems to update fairly often (including once already since this PR was opened). I know this technically mean that we could update to a version with breaking changes, but I've been under the impression that clippy basically does incremental versioning (i.e. that the two leading zeros are superfluous and only present because crates requires a semver-looking version). Plus, the worst case scenario here is that our travis build is temporarily broken; this won't case any errors for users of the library.